oddt/utils.py

```diff
@@ -64,3 +65,25 @@ def check_molecule(mol,
 
     if non_zero_atoms and len(mol.atoms) == 0:
         raise ValueError('Molecule "%s" has zero atoms.' % mol.title)
+
+
+def compose_iter(iterable, funcs):
```

Comment:
Consider using `*funcs` instead. I prefer `fun(a, f1, f2, f3)` than `fun(a, (f1, f2, f2))`.

Reply:
If we did we couldn't make a partial of it without first positional argument, like now.

```python
composed_f1_f2_f3 = partial(compose_iter, funcs=[f1, f2, f3])
composed_f1_f2_f3(range(3))
```